FastBank.Domain/MessageStatuses.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace FastBank.Domain
+{
+    public enum MessageStatuses
+    {
+        Sent,
+        Delivered,
+        Accepted,
+        Replied
+    }
```

Comment:
Put this in Message.cs. It is related to the message class and it is good to be there.